backends/xnnpack/runtime/XNNCompiler.cpp

```diff
@@ -630,7 +630,11 @@ Error defineConvertNode(
       subgraph_ptr,
       remapped_ids.at(graph_node->input_id()),
       remapped_ids.at(graph_node->output_id()),
+#ifdef ENABLE_XNNPACK_KLEIDI
+      0x00000080);
```

Comment:
what is this magic?

Reply:
XNNPACK folk didn't make this available through xnnpack.h. They actually do the same with mediapipe LOL:

https://github.com/google-ai-edge/mediapipe/blob/cae031ac4ad34cf45a2f29f02615beb049de0e49/mediapipe/tasks/cc/genai/inference/utils/xnn_utils/graph_builder.cc#L384